server/liveblog/blogs/blogs.py

```diff
@@ -100,8 +103,8 @@ def send_members_email(recipients, user_name, doc, title, url):
     admins = app.config['ADMINS']
     app_name = app.config['APPLICATION_NAME']
     subject = render_template("invited_members_subject.txt", app_name=app_name)
-    text_body = render_template("invited_members.txt", link=url, title=title)
-    html_body = render_template("invited_members.html", link=url, title=title)
+    text_body = render_template("invited_members.txt", app_name=app_name, link=url, title=title)
+    html_body = render_template("invited_members.html", app_name=app_name, link=url, title=title)
```

Comment:
would it be great to put all the features relative to membership in the same module ?
Otherwise we should rename this one by "request_membership" to avoid confusion